src/main/java/com/metamx/emitter/service/ServiceEmitter.java

```diff
@@ -27,8 +27,8 @@
 
 public class ServiceEmitter implements Emitter
 {
-  private final ImmutableMap<String, String> serviceDimensions;
-  private final Emitter emitter;
+  protected final ImmutableMap<String, String> serviceDimensions;
```

Comment:
I use serviceDimensions not only in tests but in the CloudServiceEmitter too. Should I add this annotation in that case?
I've returned private modifier for emitter.